package.json

```diff
@@ -78,7 +78,8 @@
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-test-renderer": "^16.13.1",
-    "rimraf": "^3.0.2"
+    "rimraf": "^3.0.2",
+    "uuid": "^8.3.0"
```

Comment:
why is this needed?

Reply:
I think this is added during debug, it's not needed. I'll push one commit again.